src/twisted/internet/test/test_defer_await.py

```diff
@@ -63,6 +69,29 @@ async def run2():
         res = self.successResultOf(d)
         self.assertEqual(res, "foo")
 
+    def test_coroFnToDeferredFn(self):
```

Comment:
This needs a docstring.

For the test, I was expecting to see an explicit triggering of "bar" callback.

So that we can have something like this

```
d = run()
# Check that d is a deferrred, but without a final result
bar_deferred.callback("bar")
# Check that `d` has now a result.
```

Reply:
> For the test, I was expecting to see an explicit triggering of "bar" callback.

I copied the previous test. I'd like to refactor all these tests to use `@coroFnToDeferred`

Reply:
```suggestion
    def test_coroFnToDeferredFn(self):
        """
        L{coroFnToDeferredFn} will wrap a coroutinefunction with a
        function that returns a L{Deferred}.
```